randovania/gui/ui_files/preset_trick_level.ui

```diff
@@ -86,6 +86,48 @@
           </property>
          </widget>
         </item>
+        <item>
+         <widget class="QGroupBox" name="dangerous_actions_group">
+          <property name="title">
+           <string>Allow dangerous actions</string>
+          </property>
+          <layout class="QVBoxLayout" name="verticalLayout_5">
+           <item>
+            <widget class="QComboBox" name="dangerous_combo">
+             <item>
+              <property name="text">
+               <string>Randomly</string>
+              </property>
+             </item>
+             <item>
+              <property name="text">
+               <string>Last Resort</string>
+              </property>
+             </item>
+             <item>
+              <property name="text">
+               <string>Never</string>
+              </property>
+             </item>
+            </widget>
+           </item>
+           <item>
+            <widget class="QLabel" name="cutscene_label">
+             <property name="text">
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Randomly&lt;/span&gt;: Items can be placed behind nodes that would require said item to escape.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Last Resort&lt;/span&gt;: Only allows dangerous actions if no other option is available for progression.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Never&lt;/span&gt;: Will never allow dangerous actions to happen logically.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Maybe a description of dangerous actions, which is the "moving past lock without the appropriate item" but also "doing something that can only be done once".